odk1-src/form-functions.rst

```diff
@@ -0,0 +1,719 @@
+      :header: type, name, label, calculation
+
+      begin_repeat, child_questions, Questions about child
+      text, child_name, Child's name
+      integer, child_age, Child's age
+      end_repeat
+      calculate, age_of_oldest_child, , max(${child_age})
+
+.. function:: min(nodeset)
+
+  Returns the smallest member of :arg:`nodeset`.
+
+  .. container:: details 
+   
+    .. rubric:: XLSForm
+
+    .. csv-table:: survey
+      :header: type, name, label, calculation
+
+      begin_repeat, child_questions, Questions about child
+      text, child_name, Child's name
+      integer, child_age, Child's age
+      end_repeat
+      calculate, age_of_youngest_child, , min(${child_age}) 
+
+.. _string-functions:
+  
+Strings
+--------
+
+.. _string-comparison-functions:
+
+Searching and matching strings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+.. function:: regex(string, expression)
+
+    Returns ``True`` if :arg:`string` is an *exact and complete* match for :arg:`expression`.
+
+  .. seealso:: :doc:`form-regex`
+
+  .. container:: details
+    
+    .. include:: incl/form-examples/regex-middle-initial.rst
+
+
+.. function:: contains(string, substring)
+
+  Returns ``True`` if the :arg:`string` contains the :arg:`substring`.
+
+.. function:: starts-with(string, substring)
+
+  Returns ``True`` if :arg:`string` begins with :arg:`substring`.
+
+.. function:: ends-with(string, substring)
+
+  Returns ``True`` if the :arg:`string` ends with :arg:`substring`.
+
+
+.. function:: substr(string, start[, end]) 	
+
+  Returns the substring of :arg:`string` beginning at the index :arg:`start` and extending to (but not including) index :arg:`end` (or to the termination of :arg:`string`, if :arg:`end` is not provided). Members of arg:`string` are zero-indexed.
+  
+.. function:: string-length(string)
+
+  Returns the number of characters in :arg:`string`.
+  
+.. _string-combination-functions:
+  
+Combining strings
+~~~~~~~~~~~~~~~~~~  
+
+.. function:: concat(arg [, arg [, arg [, arg [...]]]])
+
+  Concatenates one or more arguments into a single string. If any :arg:`arg` is a :term:`nodeset`, the values within the set are concatenated into a string.
+
+  
+.. function:: join(separator, nodeset)
+
+  Joins the members of :arg:`nodeset`, using the string :arg:`separator`.
+
+.. _string-conversion-functions:
+  
+Converting to and from strings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. function:: boolean-from-string(string)
+
+  Returns ``True`` if :arg:`string` is "true" or "1".
+  Otherwise, ``False``.
+
+
+.. function:: number(string)
+
+  Converts a :arg:`string` of digits into a number value.
+
+.. function:: string(arg)
+
+   Converts :arg:`arg` to a string.
+
+.. _math-functions:
+  
+Math 
+------
+
+.. _number-functions:
+
+Number handling
+~~~~~~~~~~~~~~~~~
+
+.. function:: round(number, places)
+
+  Rounds a decimal :arg:`number` to some number of decimal :arg:`places`.
+
+.. function:: int(arg) 	
+
+  Converts :arg:`arg` to an integer.
```

Comment:
To be explicit, I might say "truncates the fractional part of a decimal value to make it an integer"